packages/web-components/src/components/cbp-toast/cbp-toast.scss

```diff
@@ -74,8 +73,8 @@ cbp-toast {
   .cbp-toast-sidebar{
     display: flex;
     padding: var(--cbp-space-4x);
-    color: var(--cbp-toast-sidebar-icon);
-    background-color: var(--cbp-toast-sidebar);
+    color: var(--cbp-toast-color-icon-sidebar);
+    background-color: var(--cbp-toast-color-bg-sidebar);
     border-top-left-radius: 5px;
```

Comment:
we should have tokens for border radius.

Reply:
updated to use the --cbp-border-radius-softer token for border radius